src/python/structurefinder/pybernyopt.py

```diff
@@ -0,0 +1,36 @@
+from nwchemex import chemist
+from berny import Berny, geomlib, optimize
+from berny.solvers import MopacSolver
+
+# Molecule for testing
+#h2 = chemist.Molecule()
+#a1       = chemist.Atom("H", 1, 1.0079, 0.0, 0.0, 0.0)
+#a2       = chemist.Atom("H", 1, 1.0079, 0.0, 0.0, 1.0)
+#h2.push_back(a1)
+#h2.push_back(a2)
+
+def optimize_pyberny(molecule):
+    """
+    This function takes a chemical system and uses PyBerny
+    to optimize the provided geometry (molecule) and 
+    returns the energy of the optimized system.
+    """
+    # Convert a Chemical System to an XYZ coordinate string
+    xyz = str(molecule.size()) + "\n\n" + str(molecule.nuclei)
```

Comment:
You should change `str(molecule.nuclei)` so that it loops over the coordinates of each nuclei rather than relying on the format `molecule.nuclei` chooses to print itself in.